manual/installation.tex

```diff
@@ -1038,17 +1038,14 @@ \subsection{Installing on systems with ARMv8 based processors}
 mkdir build_armclang
 cd build_armclang
 cmake -DCMAKE_C_COMPILER=armclang -DCMAKE_CXX_COMPILER=armclang++ -DQMC_MPI=0 \
-      -DCMAKE_C_COMPILER_ID=Clang -DCMAKE_CXX_COMPILER_ID=Clang \
-      -DCMAKE_CXX_COMPILER_VERSION=5.0 -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=98 \
       -DCMAKE_C_FLAGS="-march=armv8-a" -DCMAKE_CXX_FLAGS="-march=armv8-a" \
       -DLAPACK_LIBRARIES="-L$ARMPL_DIR/lib -larmpl_mp" \
       -DFFTW_INCLUDE_DIR="$ARMPL_DIR/include" \
       -DFFTW_LIBRARIES="$ARMPL_DIR/lib/libarmpl_mp.a" \
       ..
 make -j 56
 \end{shade}
-Note that armclang is recognized as an `unknown' compiler by cmake (v3.12.4) and we need to force it as clang to apply necessary flags.
-This workaround will not be necessary in the near future.
+Note that armclang is recognized as an `unknown' compiler by cmake v3.13.* and below. In this case, we need to force it as clang (use (\ishell{-DCMAKE_\{C,CXX\}_COMPILER_ID}) to apply necessary flags.
```

Comment:
Please list the "necessary flags" explicit.

Reply:
Done